### ISSUE TITLE
⌨ Notifications focus after deleting a notification

### ISSUE THREAD

**Issue by vd1992** [GCTC-NTGC/gc-digital-talent#10637]
## ⌨ Accessibility Issue

If you are tabbing through a bunch of notifications and delete one, focus goes back to the pane. 

### WCAG 2.1 Criteria

List any relevant criteria that is failing.

## 🦋 Expected Behaviour

Focus hops to another notification if present?

## 📋 Steps to Reproduce

1. Have multiple notifications
2. Tab to delete one
3. Observe focus

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/26c20f89-0377-4c4a-ae0b-31d546801399)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/aefa772d-2ef9-4b8a-af9b-9f2d3e5d4f7e)

## 🙋‍♀️ Proposed Solution

Focus goes to the next notification if it exists. And if it doesn't, let the focus do what it wants.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Focus goes to next notification after deletion if it exists, or to the pane.
